test/createAzureResource.test.ts

```diff
@@ -112,6 +115,14 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         assert.equal(createdApp.scmType, constants.ScmType.None, `Web App scmType's property value should be ${constants.ScmType.None} rather than ${createdApp.scmType}.`);
     });
 
+    test('Generate Azure CLI Script', async () => {
+        const scriptTemplate: string = await generateAzureCLIScript('linux-default.sh', 'windows-default.sh');
```

Comment:
Since these are basically relying on the same template, it's not actually testing whether or not the CLI script would work.  I'm not sure if this test is worth having unless we run the script in the CLI and verify that it works.

Maybe we can try running it after the delete command and see if we can recreate it?  Of course, a challenge would be installing the Azure CLI